fastlane/spec/helper/sh_helper_spec.rb

```diff
@@ -1,14 +1,23 @@
 describe Fastlane::Actions do
   describe "#sh" do
+    let (:mock_input) { double :input }
+    let (:mock_output) { double :output }
+    let (:mock_status) { double :status }
+    let (:mock_thread) { double :thread, value: mock_status }
+
     before do
       allow(FastlaneCore::Helper).to receive(:sh_enabled?).and_return(true)
+      allow(mock_output).to receive(:sync=).with(true)
```

Comment:
Is there a way to just ignore these? Personally I'm not a huge fan of tests that test the internal implementation instead of the expected behaviour. The important thing is that it does shell out and the command is correct, whether it calls `each` on the standard output should be irrelevant